src/programs.c

```diff
@@ -181,11 +272,61 @@ static __always_inline syscall_pattern_t ptrace_syscall_pattern(u32 request)
         .mono_ns = mono_ns,                    \
     }
 
+#define FILL_TELEMETRY_SYSCALL_EVENT(E, SP)    \
```

Comment:
Can the `\` be all at the same indent please

Reply:
Yes they can.

750d697f2242e69233ef10fb19251be28b8c7bae